build.gradle

```diff
@@ -17,6 +13,14 @@ buildscript {
     }
 }
 
+plugins {
+    id 'io.ehdev.gradle.dependency-resolve' version '0.0.24'
```

Comment:
it allows me to call ./gradlew resolveDependencies to download dependencies and cache them. works for everything except scala